OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -101,8 +101,7 @@ public void Tick(Actor self)
 		void Announce(Actor self, TraitPair<AnnounceOnSeen> announce)
 		{
 			// Audio notification
-			if (self.World.LocalPlayer != null)
```

Comment:
I think you'd need to undo this change and then change this check to `if (self.World.LocalPlayer == self.Owner)`